frontend/app/src/data/constraints.ts

```diff
@@ -0,0 +1,15 @@
+export class Constraints {
```

Comment:
Is this class used in the code in this PR? or is it in preparation for another?

Reply:
sorry my mistake, will remove it (yes it is for another)